modules/geo-layers/src/mesh-layer/mesh-layer-fragment.glsl.ts

```diff
@@ -22,21 +19,21 @@ void main(void) {
 
   fragColor = vColor * pbr_filterColor(vec4(0));
   geometry.uv = pbr_vUV;
-  fragColor.a *= opacity;
+  fragColor.a *= layer.opacity;
 
 #else
 
   geometry.uv = vTexCoord;
 
   vec3 normal;
-  if (flatShading) {
+  if (simpleMesh.flatShading) {
```

Comment:
Nit: is our object of attention called `mesh` in some files and `simpleMesh` in others? Possible to align? Or a good reason?

Reply:
Well, the "good" reason is that we have `SimpleMeshLayer` in `mesh-layers` and `MeshLayer` which extends it in `geo-layers`. I'm not aware of the history of why we have two layers, @Pessimistress would it be a good idea to combine the two? At any rate, I'm not going to address this as part of the UBO work